File: AppTracerbility/lib/models/ProductTransactions.dart
class ProductTransactions{
  int id;
  String idProduct;
  int idLocation;
  String detail;
  String dateChecked;
  String idUser;

  ProductTransactions({this.id, this.idProduct, this.idLocation, this.detail, this.dateChecked, this.idUser});

  factory ProductTransactions.fromJson(Map<String, dynamic> map){
    return ProductTransactions(
      id : map['idProductTransactions'],
      idProduct: map['idProduct'],
      idLocation: map['idLocation'],
      detail: map['detail'],
      dateChecked: map['dateChecked'],
      idUser: map['idUser'],
    );
  }

  Map<String, dynamic> toJson(){
    return{
      // 'idProductTransactions':id,
      'idProduct':idProduct,
      'idLocation':idLocation,
      'detail':detail,
      'dateChecked':dateChecked,
      'idUser':idUser,
    };
  }
}

File: WebTracerbility/Models/ProductTransactions.cs
using System;
using System.Collections.Generic;

namespace BidPhar_services.Models
{
    public partial class ProductTransactions
    {
        public int IdProductTransactions { get; set; }
        public string IdProduct { get; set; }
        public int? IdLocation { get; set; }
        public string Detail { get; set; }
        public DateTime? DateChecked { get; set; }
        public string IdUser { get; set; }

        public virtual Location IdLocationNavigation { get; set; }
        public virtual Products IdProductNavigation { get; set; }
        public virtual Users IdUserNavigation { get; set; }
    }
}
